clients/python/tests/test_system.py

```
@@ -0,0 +1,179 @@
+    os.makedirs(ebs_dir, exist_ok=True)
+
+    config_path = str(work_dir / "test_config.yml")
+    with open(config_path, "w") as f:
+        f.write(TEST_CONFIG_YAML.format(ebs_dir=ebs_dir))
+
+    log_path = str(work_dir / "server.log")
+    procs = []
+
+    for name in ["anna-monitor", "anna-route", "anna-kvs"]:
+        bin_path = os.path.join(server_dir, name)
+        proc = subprocess.Popen(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)